tracer/test/test-applications/integrations/Samples.NetActivitySdk/Program.cs

```diff
@@ -23,6 +32,10 @@ public static async Task Main(string[] args)
 
         ActivitySource.AddActivityListener(activityListener);
 
+        RunCreateSpanLinkSpans();
```

Comment:
To make it easier to understand the snapshot changes, can you execute these two methods last?

Reply:
Disregard: They're not ordered by StartTime, I think they're ordered by resource name.